composer.json

```diff
@@ -4,14 +4,14 @@
     "license": "MIT",
     "require": {
         "php-pm/php-pm": "^2.0",
-        "symfony/http-foundation": "^3.4|^4.2.12|^5.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0",
+        "symfony/http-foundation": "^4.2.12|^5.0|^6.0",
```

Comment:
Why 4.2.12 specifically?

Reply:
I was just notified of https://github.com/advisories/GHSA-754h-5r27-7x3r. We should upgrade to `^4.1.13|^5.1.5`. I'm not sure if this version constraint can properly be handled?

I'd be happy for follow-up PR.

Reply:
Sure! I'll make a new PR with just that suggested change and see if it works out. Technically SF <5.3 is no longer supported and 5.3 drops out the end of this month. I am not sure how much of an issue that restraint is in the grand scheme of things based on that?

Reply:
Then lets just upgrade to a then-supported version

Reply:
https://github.com/php-pm/php-pm-httpkernel/pull/180 - with a comment about suggested versions.